eva_submission/eload_ingestion.py

```diff
@@ -114,3 +152,215 @@ def load_from_ena(self):
+                f.write(accession_props_template(
+                    assembly_accession=self.eload_cfg.query('submission', 'assembly_accession'),
+                    taxonomy_id=self.eload_cfg.query('submission', 'taxonomy_id'),
+                    project_accession=self.project_accession,
+                    aggregation=self.eload_cfg.query(self.config_section, 'aggregation'),
+                    fasta=self.eload_cfg.query('submission', 'assembly_fasta'),
+                    report=self.eload_cfg.query('submission', 'assembly_report'),
+                    instance_id=self.eload_cfg.query(self.config_section, 'accession', 'instance_id'),
+                    vcf_path=vcf_path,
+                    output_vcf=output_vcf,
+                    mongo_host=mongo_host,
+                    mongo_user=mongo_user,
+                    mongo_pass=mongo_pass,
+                    postgres_url=pg_url,
+                    postgres_user=pg_user,
+                    postgres_pass=pg_pass
+                ))
+            prop_files.append(str(properties_filename))
+        return prop_files
+
+    def create_variant_load_properties(self):
+        """
+        Creates properties files for the variant load pipeline, one for each VCF file.
+        """
+        prop_files = []
+        for vcf_path in self.vcfs_to_load:
+            filename = vcf_path.stem
+            properties_filename = self.project_dir.joinpath(f'load_{filename}.properties')
+            with open(properties_filename, 'w+') as f:
+                f.write(variant_load_props_template(
+                    project_accession=self.project_accession,
+                    analysis_accession=self.eload_cfg.query('brokering', 'ena', 'ANALYSIS'),
+                    vcf_path=vcf_path,
+                    aggregation=self.eload_cfg.query(self.config_section, 'aggregation'),
+                    study_name=self.get_study_name(),
+                    fasta=self.eload_cfg.query('submission', 'assembly_fasta'),
+                    output_dir=self.project_dir.joinpath(project_dirs['transformed']),
+                    annotation_dir=self.project_dir.joinpath(project_dirs['annotation']),
+                    stats_dir=self.project_dir.joinpath(project_dirs['stats']),
+                    db_name=self.eload_cfg.query(self.config_section, 'database', 'db_name'),
+                    vep_species=self.get_vep_species(),
+                    vep_version=self.eload_cfg.query(self.config_section, 'variant_load', 'vep', 'version'),
+                    vep_cache_version=self.eload_cfg.query(self.config_section, 'variant_load', 'vep', 'cache_version')
+                ))
+            prop_files.append(str(properties_filename))
+        return prop_files
+
+    def get_mongo_creds(self):
+        properties = get_properties_from_xml_file('production', self.settings_xml_file)
+        mongo_host = str(str(properties['eva.mongo.host']).split(',')[0]).split(':')[0]
```

Comment:
Please use the official [Mongo URI parser](https://pymongo.readthedocs.io/en/stable/api/pymongo/uri_parser.html) to keep up with future changes (ex: MongoDB 4.0 cluster will possibly get authMechanism parameter).

Reply:
I'll definitely do this, but this reminds me of [another question](https://github.com/apriltuesday/eva-submission/blob/EVA-2278/eva_submission/eload_ingestion.py#L220) that I had - basically the only reason I need this method rather than just `get_mongo_uri_for_eva_profile` is that the accessioning pipeline needs the credentials in the properties file.  Is there a way the pipeline can get those for itself in the future?

Reply:
The pipelines were created with the goal of being installable in other environments besides EVA. Therefore, it might be tricky. But it is something that we can look into. Could you create a ticket for that in JIRA?

Reply:
Right, that makes sense.  I'm imagining something like what eva-pipeline does, with a production properties file containing the credentials and a secondary one created per submission.  But of course I don't know if that's possible for accessioning, and it's definitely a nice-to-have from my perspective.

Reply:
Either way I've created [EVA-2336](https://www.ebi.ac.uk/panda/jira/browse/EVA-2336) so we can discuss/investigate.